src/OVAL/probes/probe-api.c

```diff
@@ -1405,7 +1405,7 @@ SEXP_t *probe_item_create(oval_subtype_t item_subtype, probe_elmatr_t *item_attr
         bool    multiplied;
         int     value_i, multiply;
 
-        subtype_name = oval_subtype_to_str(item_subtype);
+        subtype_name = oval_subtype_get_text(item_subtype);
```

Comment:
Yes, that could be a problem, the  function is used also in oval_object_get_name and other places, so it might be difficult to identify what everything needs to changed to not introduce a regression.

I think I will try to introduce oval_subtype_is_valid and use the new function in all the relevant places.